web/js/containers/tour.js

```diff
@@ -165,6 +177,13 @@ class Tour extends React.Component {
       config,
       renderedPalettes,
     );
+    currentStory.steps.forEach((step) => {
+      preProcessStepLink(
```

Comment:
Is it possible to execute this step AFTER the first step loads? I am seeing significant slowdown on the first step of the story that is remediated by removing this code block.

We are also PreProcessing Step 1 here which is duplicated effort. Can we exclude step 1 from pre-processing?

Reply:
To fix the slowdown on the first step, I changed it so that we only preload the next step, rather than all steps at the beginning. This also solves the duplicated effort on step 1 pre-processing.